frontend/src/view/Aplose_presentation/aplose.tsx

```diff
@@ -0,0 +1,162 @@
+
+        return () => {
+            isMounted = false;
+        }
+    }, [fetchCollaborators]);
+
+    return (
+        <div id="aplose-page">
+
+            <PageTitle img={imgTitle} imgAlt="Aplose Page Banner">APLOSE</PageTitle>
+            <div className="section-content">
+                <h2 className="title">Welcome to the overview page for APLOSE: Annotation Platform for Ocean Sound Explorers</h2>
+                <p>
+                    APLOSE is the OSmOSE team platform for marine acoustic research. It is an open-source annotation platform that facilitates collaboration in the field of underwater acoustics.
+                    Dive into our short video presentation to find out more about APLOSE.
+                </p>
+                <div className="video-section">
+                    <video controls style={{ width: "100%" }}>
+                        <source src={video} type="video/mp4" />
+                    </video>
+                </div>
+            </div>
+
+            <div className="section-content">
+                <h2>Manual annotation of marine sounds</h2>
+                <p>
+                    Manual annotation is the action of assigning labels on spectrograms (time/frequency representation of the sound) of a dataset. A dataset is made of audio files that will be annotated by one or several annotators during an annotation campaign.
+                    These audio files can be divided into smaller audio segments, depending on the sounds of interest. An annotation task is a spectrogram and its matching audio segment associated with a set of labels from which the annotator must choose to make the annotations. Labels are tags that designate the sounds to search on the spectrogram.
+                    They can describe broad categories of sound (biophony, geophony, anthropophony), groups of species (odontocetes, black fish, etc) or more specific types of sound (air gun, blue whale D-calls, etc). Within an annotation campaign, each annotator is given a list of annotation tasks to accomplish. There are 2 classes of annotation: weak annotation,
+                    the annotator annotates the whole spectrogram from the list of available labels ; and strong annotation, the annotator draws a labeled time-frequency box around the targeted sound event.
+                </p>
+                <img src={imgGlider} alt="Glider" style={{ width: '100%', marginTop: '20px' }} />
+            </div>
+
+            <div className="section-content">
+                <h2>Platform features</h2>
+                <p>
+                    Explore the core functionalities of APLOSE through detailed screenshots that illustrate how our platform operates. Among other, APLOSE allows to:
+                </p>
+                <ul>
+                    <li>Visualize and zoom on pre-computed spectrograms :</li>
+                        <li> - Play the sound at different speeds</li>
```

Comment:
Why adding "-" to list tags ? If you want the list style to render with "-" character you can directly set `list-style-type: "-";` to the ul css